packages/server/database/migrations/20201008134420-addPlainTextContentToTask.ts

```diff
@@ -24,6 +24,22 @@ export const up = async function(r: R) {
           })
       })
       .run()
+  }
+
+  try {
+    let i = 0
+    const batchSize = 10000
+    while (true) {
+      const tasks = await r
+        .table('Task')
+        .orderBy('createdAt')
```

Comment:
ah that makes sense. @mattkrick since ids are randomly generated, do you see any potential for a race condition to miss some tasks in the backfill? e.g. if in the process of backfilling, new tasks get created and their ids have a lower sort order than the ones we're currently processing. that was my original motivation for using `createdAt` instead of `id` field to sort.

Reply:
oh that's good thinking!
maybe we add a `filter(row.hasFields('plaintextContent').not()` and do 2 iterations?

Reply:
i like this idea, i shall do that!